docs/mkdocs.yml

```diff
@@ -96,6 +96,7 @@ nav:
             - 'Account Lockout': 'authn-guide/lockout.md'
             - 'Forgot Password': 'authn-guide/pw-reset.md'
             - 'Custom Script Tutorial': 'authn-guide/customauthn.md'
+            - 'OpenDJ Replication': 'authn-guide/custom-ldap-replication.md'
```

Comment:
this is not about replication. Its about adding a user and session affinity. This should be added as an example here https://gluu.org/docs/gluu-server/4.3/developer-guide/tips-cust-script-pages/#examples so please remove this.